js/utility/mDDuElement.js

```diff
@@ -0,0 +1,6 @@
+export default function createDurationElement(duration, element) {
+  const el = document.createElement("div");
+  el.classList.add("duration");
+  el.textContent = `Completed in ${duration} days`;
```

Comment:
Can we be a bit more specific with the naming here? Instead of `el` we can name it something more meaningful.

Reply:
taskCompleteDuration is more meaningful I guess